docs/data/material/varnish/shellac/shellac.md

```diff
@@ -0,0 +1,33 @@
+---
+title: shellac
+githubLabel: 'varnish: shellac'
+---
+
+# Shellac
+
+<p class="description">`Shellac` is a pared down version of `Varnish` that lets you make a polished, AI2 branded website with HTML alone.</p>
+
+You should use Shellac for building simple, static sites. If your application is likely to be complex and include a lot of interactivity, use Varnish proper instead.
+
+## Usage
+
+Add this [css](https://cdn.jsdelivr.net/npm/@allenai/varnish2@latest/shellac.css) to your html page.
+
+## What you get
+
+All Varnish2/MUI styles expressed as CSS Variables.
```

Comment:
is there somewhere that people can go to see the full list of CSS vars? can that be linked or added here?

Reply:
done